doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/examples/ml-example.md

```diff
@@ -0,0 +1,139 @@
+(vm-ml-example)=
+
+# XGBoost-Ray on VMs
+
+:::{note}
+To learn the basics of Ray on VMs, we recommend taking a look
+at the {ref}`introductory guide<ref-cluster-quick-start-vms-under-construction>` first.
+:::
+
+
+In this guide, we show you how to run a sample Ray machine learning
+workload on AWS cloud.
```

Comment:
`AWS cloud` -> `AWS`.

We should mention that similar steps can be used to deploy on GCP or Azure.
(Later, we can add an actual cluster config for GCP.)